crates/matrix-sdk-crypto/src/session_manager/sessions.rs

```diff
@@ -313,21 +312,30 @@ impl SessionManager {
         self.failed_devices.read().unwrap().get(user_id).is_some_and(|d| d.contains(device_id))
     }
 
-    /// Receive a successful key claim response and create new Olm sessions with
-    /// the claimed keys.
+    /// This method will try to figure out for which devices a one-time key was
+    /// requested but is not present in the response.
     ///
-    /// # Arguments
+    /// As per [spec], if a user/device pair does not have any one-time keys on
+    /// the homeserver, the server will just omit the user/device pair from
+    /// the response:
+    ///     > If the homeserver could be reached, but the user or device was
+    ///     > unknown, no failure is recorded. Instead, the corresponding user
+    ///     > or device is missing from the one_time_keys result.
```

Comment:
This is supposed to be a blockquote, not a code block, right?

```suggestion
    /// the response:
    ///
    /// > If the homeserver could be reached, but the user or device was
    /// > unknown, no failure is recorded. Instead, the corresponding user
    /// > or device is missing from the one_time_keys result.
```